src/main/webapp/script.js

```diff
@@ -11,6 +11,16 @@ function onElectionListLoad(){
   }
 }
 
+function onElectionInfoLoad(){
```

Comment:
Can we split script.js into mulitple js files, where each js files only contains the functions that are needed for each webpage?

Reply:
Sure -- I can make that change right now.